docs/releases/4.0.txt

```diff
@@ -184,7 +184,10 @@ Migrations
 Models
 ~~~~~~
 
-* ...
+* Aggregate functions now support a ``default`` argument that allows specifying
+  a value to be returned when the queryset (or grouping) contains no entries.
```

Comment:
Perhaps we can mention what it replaces?

```suggestion
  a value to be returned when the queryset (or grouping) contains no entries, rather than `None`.
```